### PR TITLE
Update Neural Insights setup.py and docs

### PR DESCRIPTION
**Pull request by bmyrcha**
## Type of Change

Documentation

API not changed

## Description

Modified way of providing `neural_insights` executable, included GUI files to setup.py and added initial documentation.

## Expected Behavior & Potential Risk

Neural Insights should be successfully installed via command `python setup.py install neural_insights`.

## How has this PR been tested?

Tested by installing via `python setup.py install neural_insights` and executing `neural_insights` after instalation.

## Dependency Change?

None

### CODE REVIEW

File: neural_insights/README.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+<div align="center">
+  
+Intel® Neural Insights
```
Comment (chensuyue) [inline neural_insights/README.md:3]: We could not use `Intel@`, this name is not formal approved.

File: setup.py [hunk 2]
```diff
@@ -65,7 +75,7 @@ def fetch_requirements(path):
         include_package_data=True,
         package_data=package_data,
         install_requires=install_requires_list,
-        scripts=scripts_list,
+        entry_points=entry_points,
```
Comment (chensuyue) [inline setup.py:78]: what's the benefit of this change?

Comment (bmyrcha) [reply setup.py:78]: It is a recommended approach for cross-platform compatibility:
https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#scripts

File: neural_insights/README.md (hunk 1 above)
Comment (bmyrcha) [reply neural_insights/README.md]: Done

**Comment by chensuyue** [comment]
Some file is empty `neural_insights/docs/source/installation_guide.md`

**Pull request merged**